util/curvesToHtml/curvesToHtml.py

```diff
@@ -125,21 +127,22 @@ def readCurvesToHtml(jobs_file, withoutOffset, showpoints, htmlBrowser):
                     if not os.path.isfile(htmlFilePath):
                         print("No HTML output file was generated by curves visualization script for job: " + current_job.jobName_)
                         continue
+                    htmlFileList.append(htmlFilePath)
 
-                    # open the html file
-                    try:
-                        webbrowser.get(htmlBrowser).open_new_tab(htmlFilePath)
-                    except Exception:
-                        try:
-                            os.system(htmlBrowser + " " + htmlFilePath)
-                        except Exception as os_error:
-                            raise os_error
                     print("... End of generating curves visualization page for job <" + current_job.jobName_ + ">")
 
                 except Exception as exc:
                     print("Fail to generate curves visualization : " + str(exc))
                     sys.exit(1)
 
+    allCurves = ""
+    for ht in htmlFileList:
+        allCurves+=""+ht+" "
```

Comment:
Why removing the use of webbrowser?

Reply:
As it seems to be at the root of the problem, it does not manage correctly the opening of multiple tabs, in particular if the browser is not running before

Reply:
ok with me, I just hope this new solution works with Apple